mvp.feature

```diff
@@ -1171,27 +1209,23 @@ Feature: MVP features for NVA
 
   # Menuitems for Administrator
   @359
-  Scenario: Administrator opens User Administration
+  Scenario: Administrator opens User Administration (Part I)
     Given that the user is logged in as Administrator
     When they click the menu item User Administration
     Then they see the User Administration page
     And they see the Sections
       | Institution Administrators |
       | Curators                   |
       | Editors                    |
+      | Creators                   |
     And they see each Section contains a list of all users with matching Role affiliated with their institution
     And they see each Section a Button to add a user with the Role associated with the Section
```

Comment:
And they see **under** each Section a Button to add a user with the Role associated with the Section